src/components/Pickers/index.jsx

```diff
@@ -23,6 +23,7 @@ const Pickers = ({
       onSelection={onChange}
       selectedValue={platform}
       options={
+        // how to pass platforms in picker option?
         Object.keys(CONFIG.platforms).reduce((res, platformKey) => {
```

Comment:
Hi @armenzg, What if I pass DASHBOARD_CONFIG here instead of CONFIG and let it be the way it is. It is working fine. 
Another option is that we make platformOptions variable in Navigation component while rendering this Pickers component and pass these options to pickers.  (as per your suggestion)

Both are giving the desired results. 

This is how dashboard config looks like. Quite similar to config. 
```
const DASHBOARD_CONFIG = {
  landingPath: '/win10/overview?numDays=90',
  timeRange: 90,
  colors: ['#e55525', '#ffcd02'],
  labels: ['Firefox', 'Chrome'],
  frameworkIds: {
    talos: 1,
    raptor: 10,
    jsbench: 11,
  },
  platforms: {
    linux64: {
      label: 'Linux 64bit',
      platform: 'linux64',
      benchmarks: Linux64.benchmarks,
    },
    mac: {
      label: 'Mac OS X',
      platform: 'osx-10-10',
      benchmarks: MacConfig.benchmarks,
    },
    win7: {
      label: 'Windows 7 32bit',
      platform: 'windows7-32',
      benchmarks: Win7Config.benchmarks,
    },
    win10: {
      label: 'Windows 10 64bit',
      platform: 'windows10-64',
      benchmarks: Win10Config.benchmarks,
    },
    android: {
      label: 'Android',
      platform: 'android', // three diff platforms in android
      benchmarks: AndroidConfig.benchmarks,
    },
  },
};
```